src/utils/customHooks.tsx

```diff
@@ -1,12 +1,9 @@
 import { useSelector } from "react-redux";
-import { useGetSelfDetailsQuery } from "../services/serverApi"
 import { RootState } from "../store";
 
 export const useGetIsSuperUser = (): boolean => {
-  const { isLoggedIn } = useSelector((state: RootState) => state.global);
-  const { data } = useGetSelfDetailsQuery(undefined, {
-    skip: !isLoggedIn
-  })
-  if (data?.roles?.super_user) return true;
+  const { roles } = useSelector((state: RootState) => state.global);
+
+  if (roles?.super_user) return true;
```

Comment:
- can we change this from `super_user` to `superUser` ?

Reply:
no we can't the property `super_user` is coming from API response and we are directly setting up